### PR TITLE
Fixes #156 Add test cases for feedParser

### LINKED ISSUE #156: Create testing for feedparser
For the recent PR #37. There are no test cases available. Assigning issue to myself with help from @evlnyng.

### PR DESCRIPTION
**Pull request by c3ho**
I think the only test case missing here is one for testing an IP address, as feedParser will actually try to connect to an IP address instead and get `ECONNREFUSED (IP here)` message instead for the test case. Perhaps @evlnyng could provide some insight if she is available.

### CODE REVIEW

**Comment by c3ho** [comment]
Pretty sure I added a new line to the end of the file, but don't know if its showing up or not.

File: test/feed-parser.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const feedParser = require('../src/feed-parser');
+
+/**
+ * feedParser(feed) async function returning an object if the feed URI is correct and there are
+ * things to parse. Invalid URI or non-feed URI should throw an error.
+*/
+test('passing a valid Atom feed URI and getting title of first post', async () => {
+  const data = await feedParser('https://c3ho.blogspot.com/feeds/posts/default/-/open-source?alt=rss');
```
Comment (humphd) [inline test/feed-parser.test.js]: We can do this in the short term, but this kind of test is going to fail when you remove your blog at some point in the future.  A better option is to use a mock network request here, using something like https://www.npmjs.com/package/jest-fetch-mock.  We'd simulate getting a feed from the network, and return a prebaked XML document you can parse.

Comment (humphd) [reply test/feed-parser.test.js]: Please file another issue to mock this out later.

Comment (c3ho) [reply test/feed-parser.test.js]: Filed issue under #210

**Pull request merged**